pkg/controller/monitor/monitor_controller.go

```diff
@@ -373,18 +379,24 @@ func (r *ReconcileMonitor) Reconcile(ctx context.Context, request reconcile.Requ
 		return reconcile.Result{}, err
 	}
 
+	felixConfiguration, err := utils.GetFelixConfiguration(ctx, r.client)
+	if err != nil {
+		log.Error(err, "Error retrieving Felix configuration")
```

Comment:
Is there a reason that we log and move on as opposed to degrading and returning? Or would that create some sort of deadlock with the core controller?

Reply:
Good question, I wouldn't expect it to be problematic.

Reply:
Updated